src/Catalyst.jl

```diff
@@ -40,6 +40,7 @@ import ModelingToolkit: check_variables,
 
 import Base: (==), hash, size, getindex, setindex, isless, Sort.defalg, length, show
 import MacroTools, Graphs
+using MacroTools: striplines
```

Comment:
this function is used frequenty, and even more frequently when debugging code. Figured it made sense to import directly.